src/test/java/com/browserstack/utils/BaseTest.java

```diff
@@ -0,0 +1,31 @@
+package com.browserstack.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BaseTest {
+
+    public static WebDriver webDriver;
+
+    @Before
+    public void init() {
+        if (StringUtils.equalsIgnoreCase(System.getProperty("browserstack-local"), "true")) {
+            System.setProperty("webdriver.chrome.driver", "src/test/resources/drivers/chromedriver");
+            webDriver = new ChromeDriver();
```

Comment:
Have we tested it locally? Just check if webDriver.get("https://localhost:3000") is needed here @ansel2000 @adarshTS

Reply:
Yes- tested. Im changing the url for localTest to   webDriver.get("http://localhost:45454/");
 here: src/test/java/com/browserstack/suites/localTest/BStackLocalTest.java. When local is true